src/Hazelcast.Net/DistributedObjects/ReliableTopicEventHandler.cs

```diff
@@ -48,25 +48,47 @@ public ReliableTopicEventHandler<T> Message(Func<IHReliableTopic<T>, ReliableTop
         return this;
     }
     
-    // Disposed event is single, no chaining.
+    /// <summary>
+    /// Sets the handler which runs after the subscription disposed or canceled.
```

Comment:
```suggestion
    /// Sets the handler which runs after the subscription is terminated and no further event will be raised.
```

Reply:
and I'd add a `<remarks>` thing explaining that "terminated" means that for "reasons" (topic overload, out-of-sync events, etc) the subscription can be terminated

Reply:
Fixed at https://github.com/hazelcast/hazelcast-csharp-client/pull/825/commits/29bfc61d64d9fcf6984c77fe142aac9fa8d43173